tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -127,6 +127,12 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
 
 To connect to your TiDB cluster using SQL shell, perform the following steps:
 
+> **Note:**
+>
+> SQL shell is only available for Dedicated Tier clusters, not for Developer Tier clusters.
+>
+> If you are a Developer Tier user, you can edit and run your SQL statements in [**Playground**](/tidb-cloud/tidb-cloud-glossary.md#playground). To access **Playground**, navigate to the **Clusters** page, locate the area of your target Developer Tier cluster, click **Playground**, and select the **Edit and Run Your SQL** tab.
```

Comment:
How about moving this to L127, just under the heading?

Reply:
+1